src/main/webapp/WEB-INF/templates/search.jsp

```diff
@@ -1,4 +1,6 @@
 <%@page contentType="text/html" pageEncoding="UTF-8"%>
+<%@page import="com.google.step.snippet.data.Card"%> 
+<%@page import="org.apache.commons.lang3.StringEscapeUtils"%>
```

Comment:
this package cannot be resolved on my dev server, but it works well without importing this package

Reply:
Did you add the dependency to pom.xml?

Reply:
import seems fine on my end